lib/constructs/static-site.ts

```diff
@@ -107,7 +107,7 @@ export class StaticSite extends Construct {
                 region: 'us-east-1',
             });
 
-            domainNamesToUse = new Array<string>(customDomain!);
+            domainNamesToUse = new Array<string>(certDomain!);
```

Comment:
While this change will work for the Application site, we will run into problem for the Admin and Landing sites - as there are no separate `certDomain` passed in. It will be `undefined` for Admin and Landing sites. I think the proper fix should be `new Array<string>(certDomain ?? customerDomain!)`. Can you please confirm this? @ACoderLife

Reply:
@zpbappi-aws I have tried your suggestion and it worked. After that, my issue was that it seems the region is hard coded to us-east-1, and I was deploying everything in us-west-2.
<img width="1410" alt="Screen Shot 2022-11-10 at 10 22 42 AM" src="https://user-images.githubusercontent.com/13307605/201150968-66972b1f-123a-4833-a0fb-3bb777f0ceba.png"> What would be a good way to trigger only the SaaSAPI stack again?

Reply:
Hi @thecount1986, thank you for the confirmation on the fix suggestion. I don't quite see any hard-coding `us-east-1` region in the code. It is only for the case of `DnsValidatedCertificate`, which has to be created in that particular region - regardless of where the rest of the workload is deployed. Can you point me to where else you find the region hard-coded?

To answer your question about running the SaaSAPI stack again- if you only have changes in SaaSAPI stack code, the CDK deployment will only deploy the changes in the subsequent runs. However, if for some edge case testing, you really need to deploy SaaSAPI only, there two ways of doing that:

- Either run the cdk script in the console with defining required environment variables for the stack and pass in the stack name at the end. It may look something like: `cdk bootstrap && cdk deploy STACK_NAME`
- Or, easier way would be to comment out the codes in `/bin/app.ts` for the other stacks that you don't want deploying.

Reply:
"us-east-1 region in the code. It is only for the case of DnsValidatedCertificate" I think this is the only place that is hard coded btw so I guess is ok. I was able to deploy vnext successfully switching using the fix you recommended. However, I have to switch to us-east-1 region. No facing problems while provisioning my first tenant. I have created a new issue for it  and started looking into it https://github.com/aws-samples/aws-saas-factory-eks-reference-architecture/issues/68

Reply:
Thank you @thecount1986 for confirming. 
@ACoderLife can you please make the above-mentioned changes into the PR? Since it is already confirmed, we can merge it in as soon as push the changes.

Reply:
@ACoderLife actually, you have given the necessary permission for me to modify the PR. Thank you for that and your contribution as always. I have made the changes and will merge it in.